docs/4_contributing/developer-guide.md

```diff
@@ -1,70 +1,68 @@
----
-title: Developer Guide
-toc: true
-weight: 710
----
 # Developer Guide
 
 Welcome to the Crossplane Developer Guide!
 
 ## Overview
 
-Infra Stacks like [stack-gcp][stack-gcp], [stack-aws][stack-aws], and
-[stack-azure][stack-azure] extend Crossplane to support managed service
-provisioning (databases, caches, buckets), secure connectivity (VPCs, subnets,
-peering, ACLs, secrets), and provisioning managed Kubernetes clusters on demand
-to further isolate the blast radius of applications.
+Infrastructure providers like [provider-gcp][provider-gcp],
+[provider-aws][provider-aws], and [provider-azure][provider-azure] extend
+Crossplane to support managed service provisioning (databases, caches, buckets),
+secure connectivity (VPCs, subnets, peering, ACLs, secrets), and provisioning
+managed Kubernetes clusters on demand to further isolate the blast radius of
+applications.
 
-Infra Stacks are typically pre-built and published to the [Stacks
-registry][stack-registry], where they can be installed by a cluster
-administrator using a [`ClusterStackInstall`][stack-install-docs] kind via the
-Kubernetes API or with the [`stack install`][crossplane-cli-usage] command.
+Infrastructure providers are typically pre-built and published to the [Docker
+Hub][crossplane-docker], where they can be installed by a cluster administrator
+using a [`ClusterStackInstall`][stack-install-docs] kind via the Kubernetes API
+or with the [`stack install`][crossplane-cli-usage] command.
 
-App Stacks depend on Infra Stacks like [stack-gcp][stack-gcp],
-[stack-aws][stack-aws], or [stack-azure][stack-azure] to provide the managed
-services they depend on via the Kubernetes API.
+Application stacks depend on infrastructure provider like
+[provider-gcp][provider-gcp], [provider-aws][provider-aws], or
+[provider-azure][provider-azure] to provide the managed services they depend on
+via the Kubernetes API.
 
-App Stacks may also be pre-built and published to the [Stacks
-registry][stack-registry] where they can be deployed by application teams using
-a [`StackInstall`][crossplane-cli-usage] kind via the Kubernetes API or with
-the [`stack install`][crossplane-cli-usage] command.
+Application stacks may also be pre-built and published to the [Docker
+Hub][stack-registry] where they can be deployed by application teams using a
```

Comment:
Looks like [Docker Hub] still points to a link named [stack-registry].

Reply:
Gotcha, this was a straight copy & paste so this must have been leftover